boot/freeldr/freeldr/include/fs/ntfs.h

```diff
@@ -115,6 +117,8 @@ typedef struct
     ULONG        BytesAllocated;
     ULONGLONG        BaseMFTRecord;
     USHORT        NextAttributeInstance;
+    USHORT        Padding;                     // Align to 4 UCHAR boundary (NTFS 3.1+ (Windows XP and above))
```

Comment:
If this is required, it makes sense to validate it with C_ASSERT (So it cannot be broken accidentally)

Reply:
As far as I can see, we can safely remove packed attribute from all structures here (except NTFS_BOOTSECTOR), and there will be no need in padding fields